doc/tech/usecases.md

```diff
@@ -2,23 +2,47 @@
 # CNTT Use Cases
 
 ## Table of Contents
- * [5.1 Core](#5.1)
- * [5.2 Edge](#5.2)
+ * [5.1 Edge](#5.1)
+   * [5.1.1 Executive Summary](#5.1.1)
+   * [5.1.2 Objective](#5.1.2)
+   * [5.1.3 Approach & Scope](#5.1.3)
+   * [5.1.4 Principle](#5.1.4)
+   * [5.1.5 Use cases](#5.1.5)
+   * [5.1.6 Terminologies](#5.1.6)
+   * [5.1.7 SDO interlocking](#5.1.7)
+
    
 <a name="5.1"></a>
-## 5.1 Core
+## 5.1 Edge
 
+<a name="5.1.1"></a>
+### 5.1.1 Executive Summary
 
-<a name="5.1"></a>
-## 5.1 Edge
+Edge computing considered one of disruptive  technology should be considered by any Digital Service provider (DSP) as it is tightly linked with a lot of use cases that can monetize the 5G adoption , therefor a lot of CSP , Public  cloud , content provider and application developer put focus on Edge 
```

Comment:
Do you need to define edge computing?

Wikipedia:
Edge computing is a distributed computing paradigm which brings computation and data storage closer to the location where it is needed, to improve response times and save bandwidth.

Edge + 5G is one combination but not necessarily the only one. Does edge help monetize 5G or 5G helps monetize Edge by enabling the low latency needs.

Reply:
The sentences within the Executive Summary need to be rephrased.

Reply:
@pgoyal01 , personally i don't like to go into the definition of the edge as it will endless discussion :) , but what you mention is fair, we can add it 


for Edge - 5G, well , 5G is service-oriented technologies it is not more about data speed ( although eMBB is defined ) so from MHO 5G need Edge to achieve 5G promises  

@Karimrabie , can comment as well

Reply:
Yes. Basically Edge is a main ingredient for 5G monetization.

Multiple 5G business use cases can’t be fulfilled by existing National/Regional PoPs due to latency constraints and other factors.